Source/WebCore/crypto/keys/CryptoKeyOKP.cpp

```diff
@@ -116,20 +116,23 @@ RefPtr<CryptoKeyOKP> CryptoKeyOKP::importJwk(CryptoAlgorithmIdentifier identifie
         break;
     }
 
-    if (!keyData.d.isNull()) {
-        // FIXME: Validate keyData.x is paired with keyData.d
-        auto d = base64URLDecode(keyData.d);
-        if (!d)
-            return nullptr;
-        return create(identifier, namedCurve, CryptoKeyType::Private, WTFMove(*d), extractable, usages);
-    }
+    if (keyData.kty != "OKP"_s)
+        return nullptr;
 
     if (keyData.x.isNull())
         return nullptr;
 
     auto x = base64URLDecode(keyData.x);
     if (!x)
         return nullptr;
+
+    if (!keyData.d.isNull()) {
+        auto d = base64URLDecode(keyData.d);
+        if (!d || !platformCheckPairedKeys(identifier, namedCurve, WTFMove(*d), WTFMove(*x)))
```

Comment:
The vectors shouldn't be moved into the `platformCheckPairedKeys()` since that indicates the Vector objects will be moved out of. The `platformCheckPairedKeys()` should just operate on normal const references.

Reply:
done